cli/pom.xml

```diff
@@ -28,6 +28,11 @@
             <artifactId>quarkus-tekton</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Most probably this is already comming in transitelvely from `quarkus-tekton-deployment`.

Reply:
If true, we should remove this.